### ISSUE TITLE
Write tests for the polling routes.

### ISSUE THREAD

**Issue by sarthakkundra** [sarthakkundra/polling-app#9]
The app currently has many routes related to the questions, all of them needs thorough testing before proceeding. Write tests for all the mentioned routes below with failing (wherever possible) and passing cases.

- [ ] Show all polls.

- [ ] Create a poll.

- [ ] Get polls of a specific user.

- [ ] Find a specific poll.

- [ ] Delete a poll.

- [ ] Cast a vote in one of the polls.

Please attach a screenshot of all the test cases passing with the PR.

**Comment by khushishikhu** [comment]
may i work on this

**Comment by sarthakkundra** [comment]
@khushishikhu Go for it! Make sure to use Jest as your testing framework.

**Comment by sarthakkundra** [comment]
@khushishikhu Please provide with updates either here or on the Gitter channel.

**Comment by khushishikhu** [comment]
@sarthakkundra  yeah sure, sorry for not updating you but i am still working on the issue

**Comment by khushishikhu** [comment]
@sarthakkundra do i have to write jest test for react

**Comment by sarthakkundra** [comment]
@khushishikhu No, you just have to write tests for the back-end API (Nodejs)

**Comment by sarthakkundra** [comment]
@khushishikhu Still no update. You can ask your doubts if you have any

**Comment by khushishikhu** [comment]
@sarthakkundra  actually i was just confused that  i should do testing with jest supertest. sorry for not updating you. will make the PR shortly

**Comment by khushishikhu** [comment]
@sarthakkundra  can you provide me any reference for writing test.
